README.md

```diff
@@ -387,6 +388,29 @@ The option `--seed` fixes the random generator seed. It should not
 generally be used except to reproduce a particular Bitmask
 trace.
 
+### Cancellation Backend (libinterflop_cancellation.so)
+
+The Cancellation backend implements an automatic cancellation detector at
+runtime. It is founded on bitwise operations to detect cancellation faster than
+in other backend. If a cancellation is detected then the backend applies noise on
+the cancelled part with the model of noise from the MCA backend. The backend
+additional cost of runtime time is constant and predetermined for each operation
+performed.
+
+This backend can be used with three options.
+
+The option `--tolerance` sets the tolerance within the backend will trigger a
+cancellation. By default tolerance is set to 1.
+
+The option `--warning` warns on the standard output stream when a cancellation is
+triggered by the backend.
+
+The option `--seed` fixes the random generator seed. It should not generally be
+used except if one to reproduce a particular MCA trace.
```

Comment:
Could you use the same format as for the other backends to present the options?

Reply:
Done. But I have seen that the format in the current output of option `--h` isn't homogeneous in other backends. Some options description have a capital letter to begin and some don't.